plugins/languages/rustaceanvim.nix

```diff
@@ -0,0 +1,271 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.rustaceanvim;
+in {
+  meta.maintainers = [maintainers.GaetanLepage];
+
+  options.plugins.rustaceanvim =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "rustaceanvim";
+
+      package = helpers.mkPackageOption "rustaceanvim" pkgs.vimPlugins.rustaceanvim;
+
+      rustAnalyzerPackage = mkOption {
+        type = with types; nullOr package;
+        default = pkgs.rust-analyzer;
+        description = ''
+          Which package to use for `rust-analyzer`.
+          Set to `null` to disable its automatic installation.
+        '';
+        example = null;
+      };
+
+      tools = {
+        executor =
+          helpers.defaultNullOpts.mkEnumFirstDefault
+          ["termopen" "quickfix" "toggleterm" "vimux"]
+          "How to execute terminal commands.";
```

Comment:
I think this can be a function: https://github.com/mrcjkb/rustaceanvim/blob/a095a1f9397310d108ead239edc068b2624695b6/lua/rustaceanvim/executors/init.lua#L13-L14

Reply:
Indeed, changed.